### PR TITLE
add current talk title to live stream

### PR DESCRIPTION
**Pull request by master-lincoln**
Add current talk title if present:

live-title

### CODE REVIEW

**Comment by TobiX** [comment]
Hmmmm... Was quite reluctant to add something like this, since there is no way to refresh listings as long as the user doesn't navigate, so the information can get stale...

**Comment by master-lincoln** [comment]
I my opinion going back and forth in the plugin to refresh is less of an annoyance than having to open the timetable or streaming website to check what's currently running.
I don't think stale information here would be too confusing to users, but maybe that's just me.

File: resources/lib/stream.py [hunk 1]
```diff
@@ -35,6 +35,8 @@ def __init__(self, json, group=''):
         if len(group) > 0:
             self.display = group + ": " + self.display
 
+        self.current_talk_title = maybe_json(maybe_json(maybe_json(json, 'talks', {}), 'current', {}), 'title', '')
```
Comment (TobiX) [inline resources/lib/stream.py:38]: This looks a bit icky, but can probably be fixed later...

**Comment by TobiX** [comment]
Thanks for the contribution!

**Pull request merged**